include/z64pause.h

```diff
@@ -63,22 +63,22 @@ typedef enum PauseState {
     /*  5 */ PAUSE_STATE_OPENING_2, // Finish some animations for opening the menu.
     /*  6 */ PAUSE_STATE_MAIN, // Pause menu ready for player inputs.
     /*  7 */ PAUSE_STATE_SAVE_PROMPT,  // Save prompt in the pause menu
-    /*  8 */ PAUSE_STATE_8,
-    /*  9 */ PAUSE_STATE_9,
-    /* 10 */ PAUSE_STATE_10,
-    /* 11 */ PAUSE_STATE_11,
-    /* 12 */ PAUSE_STATE_12,
-    /* 13 */ PAUSE_STATE_13,
-    /* 14 */ PAUSE_STATE_14,
-    /* 15 */ PAUSE_STATE_15,
-    /* 16 */ PAUSE_STATE_16,
-    /* 17 */ PAUSE_STATE_17,
+    /*  8 */ PAUSE_STATE_GAME_OVER_REQUEST,
```

Comment:
Not sure request totally fits here. To me when I hear request Im expecting a request to be sent somewhere, processed, and then a response sent back. This just sets a couple things and moves on to the next stage instantly.

My gut instinct would be to call this "init", but I see that state 10 is also called init, and it has more symmetry with the pause menu's init. So maybe just "_START", idk. thoughts @Dragorn421

Reply:
normal pause counterpart is currently called `PAUSE_STATE_WAIT_LETTERBOX`, thus might `PAUSE_STATE_GAME_OVER_WAIT` fit here?

there's no
```
// z_kaleido_scope_call.c
if (pauseCtx->state == PAUSE_STATE_WAIT_LETTERBOX) {
```
for game over variant

Reply:
Its called "wait letter box" because it is waiting for this condition to be met 
`if (Letterbox_GetSize() == 0) {`
Youll notice that in the game over variant there is no condition like that, it doesnt have to wait for anything. Its just initializing a few things right away and then continuing.

Reply:
Im not sure symmetry is strictly required here. In the pause case, this state is spending most of its time waiting for the letterbox state to update appropriately. I think it makes sense to name if after this. Meanwhile, the gameover equivalent does not have to wait for anything in this manner.

More opinions welcome though, I believe @Dragorn421 you named the pause ones initially

Reply:
got it